packages/dm-core-plugins/src/form/widgets/DimensionalScalarWidget.tsx

```diff
@@ -45,11 +50,18 @@ const DimensionalScalarWidget = ({
         }
         disabled={readOnly}
         onChange={(e: ChangeEvent<HTMLInputElement>) => {
-          if (e.target.value === '') setError('Value is required')
-          else setError('')
-          onChange({ ...entity, value: parseFloat(e.target.value) })
+          setError(e.target.value === '' ? 'Value is required' : '')
```

Comment:
Does this make every scalar required?
Thats probably not right

Reply:
discussed live